tests/driver_lsm6dsl/main.c

```diff
@@ -55,7 +55,7 @@ int main(void)
     }
     puts("[SUCCESS]\n");
 
-    xtimer_sleep(1);
+    ztimer_sleep(ZTIMER_MSEC, 1 * 1000);
```

Comment:
Is it a problem to use `ZTIMER_SEC` here? Does the simultaneous use of `ztimer_msec` and `ztimer_sec` require more resources?

Reply:
Yes, an initial backend time structure is allocated so there is more overhead, also one needs to be wary when mixing different timers as #16698 details, for `sleep` this is of course not an issue. But I would say that its preferable to:

- only use `ztimer_msec` if possible (as can be ran with rtt backend and therefore sleep)

Pulling in `ztimer_sec` makes sense IMO when you need longer timestamping. Maybe others desagree.

Reply:
I agree, only using ztimer_msec if possible would be best.

Reply:
Ok, I see, mixing usec and msec timer is a bad idea. However, it is difficult for an application to control this if it uses multiple drivers that require different timing resolutions.